js/src/views/edit-attribute-field-attachment.js

```diff
@@ -87,46 +123,37 @@ var editAttributeFieldAttachment = sui.views.editAttributeField.extend( {
 	 */
 	_renderPreview: function( attachment ) {
 
-		var $thumbnail = jQuery('<div class="thumbnail"></div>');
+		var $thumbnail = $( this.previewTemplate( attachment.toJSON() ) );
 
-		if ( 'image' !== attachment.type ) {
+		$thumbnail.appendTo( this.$container );
 
-			jQuery( '<img/>', {
-				src: attachment.icon,
-				alt: attachment.title,
-			} ).appendTo( $thumbnail );
-
-			jQuery( '<div/>', {
-				class: 'filename',
-				html:  '<div>' + attachment.title + '</div>',
-			} ).appendTo( $thumbnail );
+		attachment.on( 'remove', function() {
+			$thumbnail.remove();
+		} );
 
-		} else {
+	},
 
-			attachmentThumb = (
-				typeof attachment.sizes !== 'undefined' &&
-				typeof attachment.sizes.thumbnail !== 'undefined'
-			) ? attachment.sizes.thumbnail : _.first( _.sortBy( attachment.sizes, 'width' ) ) || {};
+	getValue: function() {
 
-			jQuery( '<img/>', {
-				src:    attachmentThumb.url || attachment.url,
-				width:  attachmentThumb.width,
-				height: attachmentThumb.height,
-				alt:    attachment.alt,
-			} ) .appendTo( $thumbnail );
+		var value = this.model.get( 'value' );
 
+		if ( ! ( value instanceof Array ) ) {
+			value = value.split( ',' );
 		}
 
-		$thumbnail.find( 'img' ).wrap( '<div class="centered"></div>' );
-		this.$container.append( $thumbnail );
-		this.$container.toggleClass( 'has-attachment', true );
+		value = value.map( function( id ) {
+			return parseInt( id, 10 );
+		} );
+
+		value = value.filter( function( id ) {
+			return id > 0;
+		} );
 
-		this.$thumbnailDetailsContainer.find( '.filename' ).text( attachment.filename );
-		this.$thumbnailDetailsContainer.find( '.date-formatted' ).text( attachment.dateFormatted );
-		this.$thumbnailDetailsContainer.find( '.size' ).text( attachment.filesizeHumanReadable );
-		this.$thumbnailDetailsContainer.find( '.dimensions' ).text( attachment.height + ' × ' + attachment.width );
-		this.$thumbnailDetailsContainer.find( '.edit-link a' ).attr( "href", attachment.editLink );
-		this.$thumbnailDetailsContainer.toggleClass( 'has-attachment', true );
+		value.sort( function( a, b ) {
+			return a < b;
+		} );
```

Comment:
I'm not totally sure about sorting the selected attachments by attachment ID, but as we don't have a sortable implementation here yet, this will do as a first introduction to the feature.